docs/user_guide/Upgrade_SIMP/Version_Maps/6.5.0_6.6.0.inc

```diff
@@ -0,0 +1,148 @@
+.. _upgrade-6.5.0-to-6.6.0:
+
+Upgrading from SIMP-6.5.0 to SIMP-6.6.0
+---------------------------------------
+
+.. contents:: :local:
+   :depth: 3
+
+.. _before-upgrading-to-6.6.0:
+
+Before Upgrading to 6.6.0
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SSSD hiera changes
+""""""""""""""""""
+There are two changes for systems that use :term:`SSSD`:
+
+
+* If LDAP and SSSD are both used, set the type of LDAP server in hiera. For EL7 servers with :term:`Openldap` set the following:
+
+  .. code-block:: yaml
+
+      simp::sssd::client:ldap_server_type: 'plain'
+
+* In previous versions of Red Hat, SSSD would fail without a LOCAL domain.  SSSD no longer requires a LOCAL domain and the :code:`simp::sssd::client` manifest no longer configures a LOCAL domain.  If you have not configured a LOCAL domain for some other reason you should remove LOCAL from the list of domains in hiera under the :code:`sssd::domains:` entry and disable the creation of the local domain by :code:`simp::sssd::client`.
+
+  .. code-block:: yaml
+
+     sssd::domains:
+       - LOCAL
```

Comment:
How does this remove `LOCAL` domain?  Looks like it is setting the domain list to `[ 'LOCAL' ]` instead?